include/session/config/groups/keys.h

```diff
@@ -292,6 +292,17 @@ LIBSESSION_EXPORT bool groups_keys_key_supplement(
         unsigned char** message,
         size_t* message_len);
 
+/// API: groups/groups_keys_current_generation
+///
+/// Returns the current generation number for the latest keys message.
+///
+/// Inputs:
+/// - `conf` -- [in] Pointer to the config object
+///
+/// Oututs:
+/// - `int` -- latest keys generation number on success; returns `-1` on failure.
```

Comment:
I don't think it's possible for this to actually error (and thus return -1), given the implementation.

Reply:
(I pushed a fix)